spec/e2e/accessibility.test.js

```diff
@@ -54,6 +54,7 @@ describe('accessibility', () => {
           'wcag22aa',
           'best-practice',
         ])
+        .exclude("div[data-touchpoints-form-id]")
```

Comment:
I'm also seeing it complain about the link at the bottom that you can click on to bring up the modal in the first place. 

![Screenshot 2025-02-04 at 13 51 27](https://github.com/user-attachments/assets/0430b4b9-c7ab-498e-9cf3-4280df055148)


Maybe this will help:
```suggestion
        .exclude("div[data-touchpoints-form-id]")
        .exclude("a#fba-button[data-open-modal]")
```

Reply:
And if you get the exact same error, maybe try `a#fba-button` without the `[data-open-modal]`? Though we'd like to be reasonably specific with this if we can